SonarQube.Client.Tests/Infra/MocksHelper.cs

```diff
@@ -32,6 +32,26 @@ internal static class MocksHelper
     {
         public const string ValidBaseAddress = "http://localhost";
 
+        /// <summary>
+        /// Sets up the HTTP message handler mock to respond to any request string 
+        /// </summary>
+        public static void SetupHttpRequest(Mock<HttpMessageHandler> messageHandlerMock, string response,
```

Comment:
New helper method that doesn't specify the expected "request" string.

If we just want to write a method that checks the expected request string has been passed then this new method is more useful than the existing one in this class.